main.go

```diff
@@ -62,7 +62,12 @@ func main() {
 	r.GET("/import/azuredevops/version", getVersionHandler)
 	r.GET("/import/azuredevops/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
-	r.Run()
+	var bindAddress string
+	var bindAddressDefined bool
+	if bindAddress, bindAddressDefined = os.LookupEnv("BIND_ADDRESS"); !bindAddressDefined {
+		bindAddress = "0.0.0.0:8080"
+	}
```

Comment:
nit: Third time reading this snippet, and now I think it'd be better if this was extracted to its own function.

```go
func getBindAddress() string {
	bindAddress, ok := os.LookupEnv("BIND_ADDRESS")
	if !ok || bindAddress == "" {
		return "0.0.0.0:8080"
	}
	return bindAddress
}
```